filetracker/cachecleaner.py

```diff
@@ -0,0 +1,199 @@
+import datetime
+import logging
+from argparse import ArgumentParser
+import time
+
+import filetracker
+
+
+logger = logging.getLogger('ftcachecleanerd')
+
+
+class CacheCleaner(object):
+    """Tool for periodically cleaning cache of the file tracker.
+       Designed to work as a daemon.
+    """
+
+    def __init__(self, cache_size_limit, cache_dirs=None,
+                 scan_interval=datetime.timedelta(minutes=10),
+                 percent_cleaning_level=50.0):
+        """
+        :param iterable cache_dirs: list of paths to
+        :class:`filetracker.Client` cache directories
+        :param int cache_size_limit: soft limit for logical files size
+        :param datetime.timedelta scan_interval: interval specifying how
+        often scan disk and optionally clean cache
+        :param float percent_cleaning_level: how many percent of
+        ``cache_size_limit`` of newest cache files do not delete during
+        cleaning cache
+        """
```

Comment:
Starałem się, by było tak samo. Poza tym, że zacząłem w złej linii, to co jeszcze jest do zmiany? Nie chcemy ```:param```?